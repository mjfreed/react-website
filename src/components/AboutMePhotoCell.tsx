import React from "react";
import { Grid, Image } from "semantic-ui-react";

interface IAboutMePhotoCell {
    imageSrc: string;
}

const AboutMePhotoCell: React.FC<IAboutMePhotoCell> = ({ imageSrc }) => {
    return (
        <Grid.Column verticalAlign="middle" style={{ justifyContent: "center", alignItems: "center", display: "flex" }}>
            <Image src={imageSrc} size="medium" />
        </Grid.Column>
    );
};

export default AboutMePhotoCell;
