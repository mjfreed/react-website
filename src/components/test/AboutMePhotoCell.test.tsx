import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router";
import ReactLogo from "../../assets/logo512.png";
import AboutMePhotoCell from "../AboutMePhotoCell";

describe("AboutMePhotoCell.tsx", () => {
    it("renders without crashing", () => {
        const { container } = render(
            <MemoryRouter>
                <AboutMePhotoCell imageSrc={ReactLogo} />
            </MemoryRouter>
        );

        expect(container).toMatchSnapshot();
    });
});
