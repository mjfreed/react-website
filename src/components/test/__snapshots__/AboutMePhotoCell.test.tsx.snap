// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMePhotoCell.tsx renders without crashing 1`] = `
<div>
  <div
    class="middle aligned column"
    style="justify-content: center; align-items: center; display: flex;"
  >
    <img
      class="ui medium image"
      src="logo512.png"
    />
  </div>
</div>
`;
